Board.js

```diff
@@ -46,12 +51,18 @@ class Board {
 
 			for (let cell of row) {
 				let td = document.createElement("td");
-				//Try to add the endship, horizontal, vertical, and NS/EW showShips
-				
 				if (showShips && cell.hasShip) td.classList.add("ship");
 				if (cell.isHit && !cell.hasShip) td.classList.add("miss");
 				if (cell.isHit && cell.hasShip) td.classList.add("hit");
-				td.addEventListener("click", e => executive.clickSpace(cell));
+				if (!final) {
+					td.addEventListener("click", e =>{
+						if (cell.hasShip) { //Testing this idea
+							this.shipSpaces--;
+							this.checkWin(executive);
+						}
+						executive.clickSpace(cell,!showShips)
```

Comment:
Also now I'm debating whether clickSpace should be in the Executive class at all or if we should just move the hit logic to the board here as well. That doesn't need to be changed before merging though.

Reply:
Noted, I'll change it later then.